components/combobox/combobox_empty-list.vue

```diff
@@ -4,9 +4,11 @@
     data-qa="dt-combobox-empty-list"
   >
     <dt-list-item
-      class="d-d-flex d-jc-center"
+      class="dt-empty-list-item d-fs-200 d-lh4 d-py4 d-px12 d-d-flex d-jc-center d-ai-center"
       role="option"
       navigation-type="none"
+      type="custom"
+      :class="itemClass"
     >
       <span>{{ message }}</span>
     </dt-list-item>
```

Comment:
Would it be better to just wrap this list item in a slot an allow them to override it however they wish?

Reply:
Yeah to be consistent with other components too, so added a slot to use here https://github.com/dialpad/dialtone-vue/pull/601/commits/ee7edea7b224a308e56286b0dd0b1cc3c0b5677b